AutoTyperGUI/Model/Chunk.cs

```diff
@@ -37,6 +37,78 @@ public KeyboardHook AutoTypeKHook
                 autoTypeKHook = value;
             }
         }
+        public static KeyboardHook createKeyboardHook(List<string> keys)
```

Comment:
Code duplication. Same function for List<string> and List<Keys> create a conversion function instead and use a list of one type. Convert on demand